### PR TITLE
DTSPO-6258 - Adding prometheus values for dev

### PR DESCRIPTION
**Pull request by endakelly**

### CODE REVIEW

File: apps/monitoring/kube-prometheus-stack/dev/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base
+  - ../../kube-prometheus-stack/dev/prometheus-values.enc.yaml
```
Comment (msl8r) [inline apps/monitoring/kube-prometheus-stack/dev/kustomization.yaml:5]: The file is in the same folder, think you just want this right?
```suggestion
  - prometheus-values.enc.yaml
```

Comment (endakelly) [reply apps/monitoring/kube-prometheus-stack/dev/kustomization.yaml:5]: ah right yeah

File: apps/monitoring/kube-prometheus-stack/prometheus-values.enc.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
```
Comment (msl8r) [inline apps/monitoring/kube-prometheus-stack/prometheus-values.enc.yaml:1]: Shouldn't this file should go into the sbox folder?

File: apps/monitoring/kube-prometheus-stack/sbox/kustomization.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../base
+  - ../../kube-prometheus-stack/sbox/prometheus-values.enc.yaml
```
Comment (msl8r) [inline apps/monitoring/kube-prometheus-stack/sbox/kustomization.yaml:5]:
```suggestion
  - prometheus-values.enc.yaml
```

File: apps/monitoring/kube-prometheus-stack/prometheus-values.enc.yaml (hunk 2 above)
Comment (msl8r) [reply apps/monitoring/kube-prometheus-stack/prometheus-values.enc.yaml:1]: or deleted

File: apps/monitoring/kube-prometheus-stack/sbox/kustomization.yaml (hunk 3 above)
Comment (endakelly) [reply apps/monitoring/kube-prometheus-stack/sbox/kustomization.yaml]: updated

File: apps/monitoring/kube-prometheus-stack/prometheus-values.enc.yaml (hunk 2 above)
Comment (endakelly) [reply apps/monitoring/kube-prometheus-stack/prometheus-values.enc.yaml:1]: This is identical to the sbox one, it was there originally before I split them into dev and sbox values. It's referred to in the [base kustomization](https://github.com/hmcts/sds-flux-config/pull/1450/files#diff-65745e18daabba5c572313d156928e768d256eecc9d206ac5ec3cf9b1e43ffc5) as a default I think but not sure if it's necessary

Comment (msl8r) [reply apps/monitoring/kube-prometheus-stack/prometheus-values.enc.yaml:1]: yeah I don't think it needs to be there, I'd delete the file and the reference

Comment (endakelly) [reply apps/monitoring/kube-prometheus-stack/prometheus-values.enc.yaml]: done

**Review by msl8r — APPROVED** [review]

**Pull request merged**